components/scream/CMakeLists.txt

```diff
@@ -58,6 +58,11 @@ if (${SCREAM_ENABLE_FPE})
   add_definitions(-DSCREAM_FPE)
 endif()
 
+prc(SCREAM_COMPILE_FLAGS)
```

Comment:
Is having the prc function really beneficial? It takes away few strokes, and I think the code is much more readable with the `MESSAGE` cmake command. If we want to have a function, then at least I would call it `print_var` or something more verbose.